sdk/storage/azure_storage_blob/src/clients/block_blob_client.rs

```diff
@@ -125,7 +125,7 @@ impl BlockBlobClient {
         options: Option<BlockBlobClientStageBlockOptions<'_>>,
     ) -> Result<Response<BlockBlobClientStageBlockResult>> {
         self.client
-            .stage_block(block_id, content_length, body, options)
+            .stage_block(&block_id, content_length, body, options)
```

Comment:
I assume that the outer `blob_id` param will change to `&[u8]` in a future PR?

Reply:
That's a question for @vincenttran-msft

Reply:
He said I could change it...